src/lib/inference/local.ts

```
@@ -1058,13 +1080,15 @@ export function getLocalProviderContainerReachabilityCheck(provider: string): st
     }
     case "ollama-local":
-      // Check the auth proxy port, not Ollama directly. The proxy listens
-      // on 0.0.0.0 and is reachable from containers; Ollama is on 127.0.0.1.
+      // Check the host port reachable from containers: raw Ollama when host
+      // loopback is reachable, otherwise the auth proxy.
       // Use -w %{http_code} (instead of -sf) so an authenticated-but-401
       // response still proves the network path works — the proxy now
       // requires a Bearer token on every endpoint (#3338) and the ephemeral
       // probe container doesn't carry one, but the goal here is connectivity
       // not authorisation.
+      const containerPort = getOllamaContainerPort();
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)